src/js/common/browserInfoHelper.js

```diff
@@ -68,7 +67,22 @@ const validateAndGetScreenDetail = (value) => {
   }
 };
 
-export const getTimezone = () => `UTC${moment().format("Z")}`;
+const getFormattedOffset = () => {
+    const nowUTC = new Date().toString();
+    const offsetIndexStart = 28;
+    const offsetLength = 33;
+    const offset = nowUTC.substr(offsetIndexStart, offsetLength);
+
+    const hourIndexStart = 0;
+    const hourLength = 3;
+    const minuteIndexStart = 3;
+    const minuteLength = 2;
```

Comment:
Didn't quite understand this logic. Is this the standard logic for parsing to UTC timezone?

Reply:
Also, it looks like we are just removing moment from here to reduce the bundle size but will still continue using moment for unit tests. Is this a standard practice? I find it a bit confusing to use it in unit tests but not in the actual code.

Reply:
Yes, this method now just essentially gets the right bits from the string (which is in a standard format that does not change) and reconstructs it into the format we require (with the `:`)

It is quite normal to have dependencies which are only used during testing and development. It is a bit unusual for moment.js to be one, but I think it makes perfect sense. What is the alternative? Essentially to rewrite this method to tests itself, which is very circular and gives me less confidence than testing against the gold standard of moment.js. 

The other benefit is that it ensures the test is environment agnostic, e.g. it doesn't care which timezone the test is being run in. To test without moment we would either tie ourselves to the environment the test is run in (e.g. is it UTC+1 or UTC + 7?) *or* mock the global variable, which makes the test quite open to being wrong, in my opinion.

So I would say there is significant upside to keeping moment as a devDependency (quality and durability of the test) with no/very limited downside (developers have to install the package to run the tests)